river/command/output.zig

```diff
@@ -52,7 +52,20 @@ pub fn sendToOutput(
     _: *?[]const u8,
 ) Error!void {
     if (args.len < 2) return Error.NotEnoughArguments;
-    if (args.len > 2) return Error.TooManyArguments;
+    const result = flags.parser([:0]const u8, &.{
+        .{ .name = "active-tags", .kind = .boolean },
+    }).parse(args[1..]) catch {
+        return error.InvalidValue;
+    };
+    if (result.args.len < 1) return Error.NotEnoughArguments;
+    if (result.args.len > 1) return Error.TooManyArguments;
+
+    var dir_arg = args[1];
+
+    // If -current-tags is set, the direction parameter will be shifted by one
+    if (args.len == 3) {
+        dir_arg = args[2];
+    }
```

Comment:
all of this is no longer needed now that you use the args parser.

Reply:
removed